website/docs/components/app-side-nav/partials/guidelines/guidelines.md

```diff
@@ -67,9 +66,16 @@ Don’t override the color of the icon within the list item or incorporate a bra
 
 #### With nested items
 
-Use `hasSubItems` to show or signify that a link has a nested level of navigation.
+Use `hasSubItems` to show or signify that a link has a nested level of navigation within the App Side Nav list. A common use case for this is when a link doesn't actually navigate to a dedicated page directly, but instead to a category of pages.
```

Comment:
> when a link doesn't actually navigate to a dedicated page directly, but instead to a category of pages.

But it does navigate to a dedicated page. 

1. Navigate to https://app.terraform.io/app/hashicorp-v2/workspaces
2. Click "Settings" (goes to a new page & updates the menu)

Reply:
I think the new illustrations and new "Don't" already go a long way towards clarifying what is meant.

So how about keeping the text the same here:
"_Use hasSubItems to show or signify that a link has a nested level of navigation._"

Then within the "don't", change the text to:
"_Don't use hasSubItems to indicate that a list item directs to a page containing a list of links within the main content are of the page itself._"

Reply:
@KristinLBradley's suggestion makes sense to me.